docs/zello-sdk-documentation/com/zello/sdk/Audio.html

```diff
@@ -265,9 +265,8 @@ <h2 title="Class Audio" class="title">Class Audio</h2>
      Sets the current <code>AudioMode</code> to <code>mode</code>.
  </p>
  <p>
-     The method is asynchronous so using <code>Audio.getMode()</code> immediatelly after calling
-     it may return the previous audio mode. <a href="../../../com/zello/sdk/Events.html#onAudioStateChanged--"><code>Events.onAudioStateChanged()</code></a> is called when
-     audio mode changes.
+     This method is asynchronous. When the mode has changed, <code>onAudioStateChanged()</code>
```

Comment:
I did that because we have the link provided with a @see tag below. That's the style that's being used in other places like Zello.beginMessage()